AsyncDisplayKit/ASEditableTextNode.mm

```diff
@@ -189,6 +191,21 @@ - (void)setOpaque:(BOOL)opaque
 @synthesize delegate = _delegate;
 
 #pragma mark -
+@dynamic textView;
+
+- (UITextView *)textView{
+  ASDisplayNodeAssertMainThread();
+  return _textView;
```

Comment:
One last thing that worries me here: if this is called before it is initialized, it's likely because a client will be attempting to set properties on the text view in an -init method.  Say that it is on the main thread, like a UIView init directly hosting an ASEditableTextNode.  The problematic pitfall is that nil messaging will mean the calls fail silently, and the developer is left wondering why their configuration is not working...